Packs/CortexXDR/Playbooks/playbook-Cortex_XDR_-_Possible_External_RDP_Brute-Force.yml

```diff
@@ -0,0 +1,3115 @@
+id: Cortex XDR - Possible External RDP Brute-Force
+version: -1
```

Comment:
<img width="1384" alt="image" src="https://user-images.githubusercontent.com/28757135/213924834-6fa33d60-bc5b-4043-a97c-deadf4e907c7.png">
Didn't we want to remove the Get entity alerts for the user? we can use the User Investigation - Generic